pkg/templates/aws/resources/elasticache_cluster.yaml

```diff
@@ -47,11 +47,16 @@ properties:
     configuration_disabled: true
     deploy_time: true
     description: The port number on which each of the cache nodes accepts connections.
+  CacheNodeAddress:
+    type: string
+    configuration_disabled: true
+    deploy_time: true
+    description: The endpoint address of a single node in the ElastiCache cluster.
```

Comment:
Why a single node and not all the nodes' addresses?

Reply:
yeah we definitely could return an array of all the addresses, i did single node just because its plug in the ref and your code will work, you dont have to split an array or anything and i figured the no touch approach is good for both our e2e suite and tutorials, etc